e2e/npm_link_package-esm/WORKSPACE

```diff
@@ -47,6 +47,10 @@ local_repository(
     path = "../rules_foo",
 )
 
-load("@rules_foo//:repositories.bzl", rules_foo_repositories = "repositories")
+load("@rules_foo//foo:repositories.bzl", rules_foo_repositories = "repositories")
 
-rules_foo_repositories()
+rules_foo_repositories(name = "foo")
+
+load("@foo//:repositories.bzl", foo_npm_repositories = "npm_repositories")
```

Comment:
Do both of these need to be named `repositories.bzl`? Would it be better to have different names?

Reply:
It would be but In this case `@foo` is just a customized `npm_translate_lock` and that currently generates a `repositories.bzl`. Suppose the name of the generated file could that customizable.

Reply:
Updated

Reply:
Yeah I like that more. It more clearly shows that one is repos while one is the npm packages of that repo 👍